README.md

```diff
@@ -51,10 +51,8 @@ nopcm_diff | minimal SWHS example, with PCM removed
 
 ## Running the example(s)
 
-Please note that if `make` has beeen used, docs are already generated automatically and can be found in build.
```

Comment:
Should this paragraph have been removed?

Reply:
This paragraph was recently added for #726, so it should not be removed.

Reply:
README.cd file has been updated. 7678ba278161aaef48071796ed0520a47767593f